src/commands/index.rs

```diff
@@ -61,7 +61,7 @@ fn add_field(index_name: &str, field_name: &str, path: &str) -> RedisResult {
     if schema.fields.contains_key(field_name) {
         Err("Field already exists".into())
     } else {
-        schema.index.create_field(field_name);
+        schema.index.create_field(field_name, 1.0, None, false);
```

Comment:
We should fix the ambiguous arguments (per your redisearch_api PR).

Reply:
:+1: